src/main/java/com/hkrsdgroup/agileproduct/DailySchedule.java

```diff
@@ -0,0 +1,160 @@
+package com.hkrsdgroup.agileproduct;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DailySchedule {
+
+    private int sleepAmount;
+    private int startDay;
+    private int endDay;
+    private String assigment1;
+    private String workout;
+    private final int breakfast = 30;
+    private final int lunch = 60;
+    private final int shortSession = 30;
+    private final int longSession = 120;
+    private final int shortBreak = 5;
+    private final int longBreak = 30;
+
+    public DailySchedule(int sleep, String assignment, String workout, int beginDayHour){
+        this.sleepAmount = sleep*60;
+        this.assigment1 = assignment;
+        this.workout = workout;
+        this.startDay = beginDayHour;
+        this.endDay = (24*60 + beginDayHour) - sleepAmount;
+    }
+
```

Comment:
@tobleroo , One more thing that wanted to mention is that in methods below, you refer directly to startDay.
It would be better if you used your method setStartDay().
This way, if you will need to change the name of property "startDay", you will not have to go through your code and search where did you used that name. Instead you will just change it in your getters and setters.